### ISSUE TITLE
lardon-metadata

### ISSUE THREAD

**Issue by YoannKermaidic** [dune-lardon/lardon#4]
Perhaps best to start an issue on the metadata =). So we can list the config files we need and their specs.

**Comment by olantwin** [comment]
I can think of three right now:

* Constant config (mostly already done by @lzambell ) for information needed for basic parsing of files, headers etc. and information that should not change
* Time-variable config for things that will change (e.g. bias voltages, channel mappings etc). Maybe this should have a higher priority/overwrite the constant config, in case something assumed to be constant starts changing...
* Local config per machine, not tracked by git: e.g. path to local data. Should overwrite info in constant config if it exists, but be optional.

@lzambell anything you want to add?

**Comment by YoannKermaidic** [comment]
One can also have a `data-sets` directory that contains lists of validated data files. Typically useful in order to loop over a bunch of files that we will use for a given analysis (like cosmics, calibration, noise, ...). If we would analyse all current TDE files, such a file list could be:
`run00300-00304-test.list`
300_1_cb
300_2_cb
301_1_cb  
301_2_cb
302_1_cb  
302_2_cb
303_1_cb  
303_2_cb
304_1_cb 
304_2_cb
It's easy to remove keys by hand if data quality tells us.

**Comment by olantwin** [comment]
> One can also have a `data-sets` directory that contains lists of validated data files. Typically useful in order to loop over a bunch of files that we will use for a given analysis (like cosmics, calibration, noise, ...). If we would analyse all current TDE files, such a file list could be: `run00300-00304-test.list` 300_1_cb 300_2_cb 301_1_cb 301_2_cb 302_1_cb 302_2_cb 303_1_cb 303_2_cb 304_1_cb 304_2_cb It's easy to remove keys by hand if data quality tells us.

I agree, but I think this is probably another, separate issue. For now let's maybe try to focus on reading/processing one file at a time. Once we can do that, this should be easy to implement/add.
